pkgs/tools/networking/hp2p/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, openmpi, python2Packages, autoconf, automake } :
+
+stdenv.mkDerivation rec {
+  pname = "hp2p";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "cea-hpc";
+    repo = "hp2p";
+    rev = "${version}";
+    sha256 = "0zvlwb941rlp3vrf9yzv7njgpj3mh4671ch7qvxfa4hq2ivd52br";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ autoconf automake python2Packages.wrapPython ];
+  buildInputs = [ openmpi ] ++ (with python2Packages; [ python numpy matplotlib plotly ]) ;
+  pythonPath = (with python2Packages; [ numpy matplotlib plotly ]) ;
```

Comment:
Do we really need python2?

Reply:
Going to check this...

Reply:
Well, yes, the python script for vizualisation is not python3 compliant...
```
[bzizou@bart:~/git/nixpkgs]$ ./result/bin/vizhp2p
  File "/nix/store/mn7llfwvwbrbfqs6zwph3p5xs2ky0in2-hp2p-3.3/bin/.vizhp2p-wrapped", line 98
    print line
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(line)?

```

Reply:
I'll try to patch it, as we now have a problem with python2:
```
[bzizou@bart:~/git/nixpkgs]$ nix-build -A hp2p
error: pyrsistent-0.17.3 not supported for interpreter python2.7
```